cogs/smallfunctions.py

```diff
@@ -0,0 +1,43 @@
+import discord
+import random
+import #new file storing priv variables
+from discord.ext import commands
+
+# Class responsible for all the small commands run by server members.
+class SmallFunctions(commands.cog):
```

Comment:
Maybe instead of SmallFunctions we have a name for the file and class that is easier to know what the purpose of it is. Maybe SmallCommands or SmallFeatures?